From 5e348707ba4bece4ad4c75a8b640f1c40cbbaa0e Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 6 Aug 2015 11:27:40 -0400
Subject: [PATCH 36/39] Ticket 48215  - update dbverify usage in main.c

Description:  Need to update dbverify usage in main.c

https://fedorahosted.org/389/ticket/48215
(cherry picked from commit c1912cdcac8319e2fe0f98f765aa935e6a8ff297)
(cherry picked from commit c842dbe3d0ee2ac28c55e31b9b8e5e5a3c5dc200)
---
 ldap/servers/slapd/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/main.c b/ldap/servers/slapd/main.c
index 922de97..4f9fbfe 100644
--- a/ldap/servers/slapd/main.c
+++ b/ldap/servers/slapd/main.c
@@ -398,7 +398,7 @@ usage( char *name, char *extraname )
 	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-N] -n backend-instance-name -a fullpath-backend-instance-dir-full\n";
 	break;
     case SLAPD_EXEMODE_DBVERIFY:
-	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-n backend-instance-name]\n";
+	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-n backend-instance-name] [-a db-directory]\n";
 	break;
 
     default:	/* SLAPD_EXEMODE_SLAPD */
-- 
1.9.3

